cadquery/occ_impl/shapes.py

```diff
@@ -1150,6 +1199,67 @@ class Solid(Shape, Mixin3D):
     a single solid
     """
     
+    @classmethod
+    def interpPlate(cls, surf_edges, surf_pts, thickness, Degree=3, NbPtsOnCur=15, NbIter=2, Anisotropie=False, Tol2d=0.00001, Tol3d=0.0001, TolAng=0.01, TolCurv=0.1, MaxDeg=8, MaxSegments=9):
```

Comment:
typo: Aniosotropie -> Anisotropy. BTW: why are the arg names capitalized?

Reply:
The original OCC function args are like this:
``BRepOffsetAPI_MakeFilling (const Standard_Integer Degree=3, const Standard_Integer NbPtsOnCur=15, const Standard_Integer NbIter=2, const Standard_Boolean Anisotropie=Standard_False, const Standard_Real Tol2d=0.00001, const Standard_Real Tol3d=0.0001, const Standard_Real TolAng=0.01, const Standard_Real TolCurv=0.1, const Standard_Integer MaxDeg=8, const Standard_Integer MaxSegments=9)``
All capitalized and using ``Anisotropie`` instead of ``Anisotropy``

I have changed it in ``cq.py`` to lowercase and anisotropy:
``def interpPlate(self, surf_edges, surf_pts=[], thickness=0, combine=True, clean=True, degree=3, nbPtsOnCur=15, nbIter=2, anisotropy=False, tol2d=0.00001, tol3d=0.0001, tolAng=0.01, tolCurv=0.1, maxDeg=8, maxSegments=9):``

In ``shapes.py`` I left the original OCC argument naming convention because ultimately ``BRepOffsetAPI_MakeFilling`` needs to be called with OCC convention. Is it OK like this?

Reply:
+1